### ISSUE TITLE
Turf v7 in BigQuery: TypeError: base intrinsic for Date.prototype.getTime exists, but the property is not available

### ISSUE THREAD

**Issue by svbatalov** [Turfjs/turf#2632]
Hello. Thanks for this great library!

I've encountered the following error message when tried to use a browserified version of Turf@7.0.0 in BigQuery:
> TypeError: base intrinsic for Date.prototype.getTime exists, but the property is not available ...

The way I bundled the library
```sh
browserify -r @turf/turf -s turf > turf7.js
```
Then used it in a BQ UDF as follows:
```sql
CREATE TEMP FUNCTION tst()
RETURNS STRING
LANGUAGE js
OPTIONS (
    library=["gs://.../turf7.js"]
) AS """
""";

SELECT tst()
```
As a workaround, I created the entry point JS file and added this line
```js
Date.prototype.getTime = () => {};
```
before requiring the library.

It looks like this issue is caused by the `deep-equal` library that references `getTime` [here](https://github.com/inspect-js/node-deep-equal/blob/main/index.js#L24).

Not sure if there are any action items for this issue -- after all, it probably should be filed against `deep-equal`, so feel free to close it.  Just wanted to log it here for anyone encountering similar issues.

Cheers,
Sergey

**Comment by smallsaucepan** [comment]
Hi @svbatalov. Yeah, deep-equal has been causing us a few problems of late, and we're in the process of removing it as a dependency.

Keep an eye on #2623 and #2631. Once they're both merged and shipped, see if that solves your issue. If the problem persists add a comment here.

**Comment by smallsaucepan** [comment]
> Keep an eye on https://github.com/Turfjs/turf/pull/2623 and https://github.com/Turfjs/turf/pull/2631

@svbatalov FYI both those PRs were released part of Turf 7.1.0. Will close this issue as resolved assuming it won't be an problem once you've upgraded. Reopen if you're still having an issue though.
